docs/pages/branding/about.tsx

```diff
@@ -522,23 +522,15 @@ function AboutContent() {
             <Button
               component={Link}
               noLinkStyle
+              passHref
```

Comment:
This is suspicious. We have tried to design `Link` so that this is not necessary. I think that it hides something wrong. I also think that it matters because it's what we encourage the community to use with Next.js.

Reply:
I will remove `passHref`, it is `true` by default in #27936